src/Magnum/Math/Test/Vector4Test.cpp

```diff
@@ -293,6 +297,24 @@ void Vector4Test::planeEquationNormalPoint() {
     CORRADE_COMPARE(eq, (Vector4{-0.9045340f, 0.3015113f, -0.3015113f, 1.658312f}));
 }
 
+void Vector4Test::strictWeakOrdering() {
+    StrictWeakOrdering<Vector4> ov4;
+    const Vector4 v4a{1, 2, 3, 4};
+    const Vector4 v4b{2, 3, 4, 5};
+    const Vector4 v4c{1, 2, 3, 5};
+    const Vector4 v4d{1, 2, 3, 4};
+
+    CORRADE_VERIFY( ov4(v4a, v4b));
+    CORRADE_VERIFY(!ov4(v4b, v4a));
+    CORRADE_VERIFY( ov4(v4a, v4c));
+    CORRADE_VERIFY(!ov4(v4c, v4a));
+    CORRADE_VERIFY( ov4(v4c, v4b));
+    CORRADE_VERIFY(!ov4(v4b, v4c));
+
+    CORRADE_VERIFY(!ov4(v4a, v4d));
+    CORRADE_VERIFY(!ov4(v4d, v4a));
```

Comment:
Thinking out loud: I was looking if you are testing also comparison with the item itself and took me a while until I realized `v4a` and `v4d` are the same. What about removing `v4d` and using `CORRADE_VERIFY(!ov4(v4a, v4a));` instead? (and similarly elsewhere?)

Yes, I know, this change would weaken the test in case the `StrictWeakOrdering` implementation would have a special case for `&a == &b`. But it's not like that, so I think it's safe.